code/code/spec/spec_objs_thief_quest.cc

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+
+#include "being.h"
+#include "comm.h"
+#include "extern.h"
+#include "obj_base_weapon.h"
+
+int thiefQuestWeapon(TBeing *victim, cmdTypeT command, const char *arg, TObj *object, TObj *) {
+  wearSlotT limb = WEAR_NOWHERE;
+
+  if (command != CMD_STAB && command != CMD_BACKSTAB && command != CMD_SLIT) return false;
+  if (command == CMD_STAB) limb = static_cast<wearSlotT>(reinterpret_cast<uintptr_t>(arg));
+  if (command != CMD_STAB && strcmp(arg, "-special-")) return false;
+  if (!victim || !object) return false;
+
+  limb = command == CMD_BACKSTAB ? WEAR_BACK
```

Comment:
I find it looks better when the ? is split into the next line.
```cpp
limb = command == CMD_BACKSTAB
    ? WEAR_BACK
    : command == CMD_SLIT
    ? WEAR_NECK
    : limb;
```

Reply:
I agree - I formatted this using the clang-format rules @wershlak set up in the repo so that might be one we need to change. I'll just reformat them manually for now.

Reply:
Personally I also favor having the statement in a single-statement `if` on its own line too, even when it's pretty short. Feels like it adds readability. Maybe that's something we could change in clang-format too.

Reply:
Resolved in https://github.com/sneezymud/sneezymud/pull/368/commits/7cd35cc3ce8966bdb70498453540f0af242e6778